src/chainparams.cpp

```diff
@@ -892,3 +899,8 @@ void UpdateDevnetLLMQChainLocks(Consensus::LLMQType llmqType)
 {
     globalChainParams->UpdateLLMQChainLocks(llmqType);
 }
+
+void UpdateLLMQTestParams(int size, int threshold)
+{
+    globalChainParams->UpdateLLMQTestParams(size, threshold);
+}
```

Comment:
nit: new line at EOF
```suggestion
}

```